src/client/dfs/obj.c

```diff
@@ -1708,6 +1726,7 @@ dfs_sync(dfs_t *dfs)
 	if (dfs->amode != O_RDWR)
 		return EPERM;
 
+	DFS_OP_STAT_INCR(dfs, DOS_SYNC);
```

Comment:
since this is a no-op, I would not increment the metric for it as it can cause confusion.